docs/doc/30-reference/30-sql/00-ddl/40-stage/06-ddl-show-stages.md

```diff
@@ -0,0 +1,23 @@
+---
+title: SHOW STAGES
+sidebar_label: SHOW STAGES
+---
+
+Returns a list of the created stages. The output list does not include the [User Stage](index.md#user-stage).
+
+## Syntax
+
+```sql
+SHOW STAGES;
+```
+
+## Examples
+
+```sql
+SHOW STAGES;
```

Comment:
Maybe we need to show user stage too.

cc @BohuTANG @wubx for ideas.

Reply:
Indeed not. The stage is user level from the users' view.